charts/budibase/templates/proxy-service-deployment.yaml

```diff
@@ -82,7 +82,9 @@ spec:
         resources:
         {{- toYaml . | nindent 10 }}
         {{ end }}
-        volumeMounts:
```

Comment:
This line wasn't doing anything, and appears to be a holdover from the initial creation of the Helm chart.